pkgs/top-level/all-packages.nix

```diff
@@ -25756,6 +25756,8 @@ with pkgs;
     php = php81;
   };
 
+  snips-sh = callPackage ../by-name/sn/snips-sh/package.nix { withTensorflow = false; };
```

Comment:
You don't need this, it's already by default

Reply:
The vet script fails if otherwise

Reply:
The new vet script would fail me otherwise.

Reply:
So while it may not be required the script doesn't understabd that.

Reply:
I believe you might have misunderstood the CI output: You can remove this line here entirely, no change to `all-packages.nix` necessary. Can you try that?

Reply:
I didn't notice it's no longer required :smile: